src/plugins/data_source_management/public/components/text_content/text_content.ts

```diff
@@ -125,7 +125,7 @@ export const ENDPOINT_URL = i18n.translate('dataSourcesManagement.createDataSour
 export const ENDPOINT_PLACEHOLDER = i18n.translate(
   'dataSourcesManagement.createDataSource.endpointPlaceholder',
   {
-    defaultMessage: 'The connection URL',
+    defaultMessage: 'Sample URL: https://connectionurl.com',
```

Comment:
Why are we declaring text like this? When we do this, the `i18n.translate` is called at compile time instead of runtime. We can declare a key value pair for the translation key and default message in this file if necessary, but lets keep the translation call in the component since we want the translation value to be fetched dynamically.

Reply:
Good catch, will change it later!